packages/braid-design-system/src/lib/components/OverflowMenu/OverflowMenu.docs.tsx

```diff
@@ -190,6 +190,49 @@ const docs: ComponentDocs = {
           </>,
         ),
     },
+    {
+      label: 'Sizes',
+      description: (
+        <Text>
+          You can customise the size of the menu via the <Strong>size</Strong>{' '}
+          prop, which accepts either <Strong>standard</Strong> or{' '}
+          <Strong>small</Strong>.
+        </Text>
+      ),
+      Example: () =>
+        source(
+          <Stack space="medium">
+            <Inline alignY="center" space="medium">
+              <Text>Standard</Text>
+              <OverflowMenu size="standard" label="Options" id="example">
+                <MenuItem id="menuItem1" onClick={() => {}}>
+                  Item 1
+                </MenuItem>
+                <MenuItem id="menuItem2" onClick={() => {}}>
+                  Item 2
+                </MenuItem>
+                <MenuItem id="menuItem3" onClick={() => {}}>
+                  Item 3
+                </MenuItem>
+              </OverflowMenu>
+            </Inline>
+            <Inline alignY="center" space="medium">
+              <Text size="small">Small</Text>
+              <OverflowMenu size="small" label="Options" id="example">
```

Comment:
These two should have unique ids
```suggestion
              <OverflowMenu size="standard" label="Options" id="size-standard">
                <MenuItem id="menuItem1" onClick={() => {}}>
                  Item 1
                </MenuItem>
                <MenuItem id="menuItem2" onClick={() => {}}>
                  Item 2
                </MenuItem>
                <MenuItem id="menuItem3" onClick={() => {}}>
                  Item 3
                </MenuItem>
              </OverflowMenu>
            </Inline>
            <Inline alignY="center" space="medium">
              <Text size="small">Small</Text>
              <OverflowMenu size="small" label="Options" id="size-small">
```